docs/_posts/GitTutorial-brief.md

```diff
@@ -0,0 +1,256 @@
+
+Git tracks your files by essentially taking a snapshot of a directory or subdirectory structure and saving it over time. The process of capturing one of these processes is called a commit.
+
+To be a little more specific, you can think of Git as working in three different areas or bins. There is (1) the workspace, (2) the repository, and (3) the index. 
+
+#### Workspace: 
+The set of directories and files that you actually operate on.
+
+#### Repository (repo)
+The set of linked commits corresponding to snapshots of the workspace at specified points in time.
+
+#### Index
+The staging area where you "set up your snapshot"; files in the index that have changed since your last commit will be updated on your next commit.
+
+### Hashes
+Each commit is given a unique label (called a hash) and is tied to the previous commit. This is created using a function which converts a set of information (i.e. your docs) into a string of letters and numbers. Hashes include a layer of security, since each relies on its parent's hash.
+
+### Diagramatic Representation
+Below is a diagram of the git process for a single file; I've named it file_1. You start by creating the file in your workspace. 
+
+![](fig/git_wd01.png)
+
+Once you are satisfied with it's progress, you decide that you want to commit your work. You add the file to the index. 
+
+![](fig/git_index01.png)
+
+Now you're ready. You can commit the file to your repository. (Notice that in the repository file_1 is shown with the hash of the commit. In reality the full hash for a git commit is much longer.)
+
+![](fig/git_repo01.png)
+
+Now, if you make some changes to file_1. Your workspace changes. 
+
+![](fig/git_wd02.png)
+
+You add the changed file_1 to the index.
+
+![](fig/git_index02.png)
+
+And again, you finally you commit. The new commit replaces the most recent commit, which moves deeper into the repository's history.
+
+![](fig/git_repo02.png)
+
+And the process repeats, on and on, continually building up your repository's history.
+
+![](fig/git_repo_many.png)
+
+## Your turn to Git started
+
+First we want to make a directory to track as the repository for this tutorial. Go somewhere in your file system and create this directory. Then navigate inside the newly created directory. (In the following code snippets, a \$ indicates a command line prompt.)
+
+```
+$ mkdir git_tutorial
+$ cd git_tutorial
+```
+
+Once you're inside, its time to initialize the repository. Intuitively, this is done with the command
+
+```
+$ git init 
+```
+
+Now, since this is likely the first time you're creating a Git repo, you may want to set up some Git configurations. Feel free to skip this step (it is optional) but if you don't do it now, Git will likely ask you for this information repeatedly in the future.
```

Comment:
Can we defer the user name set up after git prompted to do so? (If nothing is broken, don't fix it)